src/highlight/__tests__/HighlightList-test.tsx

```diff
@@ -27,28 +27,35 @@ describe('HighlightList', () => {
     const getWrapper = (props?: Partial<Props>): ShallowWrapper => shallow(<HighlightList {...defaults} {...props} />);
 
     describe('render()', () => {
+        const mockSetHoverAnnotationId = jest.fn();
         const mockSetIsListening = jest.fn();
-
-        beforeEach(() => {
-            jest.spyOn(React, 'useState').mockImplementation(() => [true, mockSetIsListening]);
-        });
+        const setupMocks = ({
```

Comment:
It seems like this will lead to further testing of the component internals rather than testing its input/output. Can we trigger these changes by simulating events, instead?